app/controllers/claims_controller.rb

```diff
@@ -3,6 +3,14 @@ class ClaimsController < BasePublicController
 
   skip_before_action :send_unstarted_claimants_to_the_start, only: [:new, :create, :signed_out]
 
+  before_action :create_session_if_skip_landing_page, if: :skip_landing_page?
+
+  def create_session_if_skip_landing_page
+    return unless journey.use_navigator?
+
+    find_journey_session || create_journey_session!
```

Comment:
```suggestion
    journey_session || create_journey_session!
```
not a blocker, but might be better to use `journey_session` as it calls `find_journey_session` and sets the `@journey_session` ivar, so saves a second call to the db if some other callback needs to use the session in this request.